src/js/oErrors.js

```diff
@@ -377,19 +382,11 @@ Errors.prototype.handleLogEvent = function(ev) {
  * @returns {Array} - An array of Elements.
  */
 Errors.prototype._getEventPath = function(event) {
-
-	// event.path is available in some browsers, most notable Chrome
-	if (event.path) {
-		// Array.prototype.slice.call coerces a NodeList to an array. Could
-		// use Array.from but it is not in the Polyfill service default set.
-		return Array.prototype.slice.call(event.path);
-	}
-
 	var path = [];
 
 	// IE backwards compatibility (get the actual target). If not IE, uses
 	// `event.target`
-	var element = window.event ? window.event.srcElement : event.target;
```

Comment:
That's an improvement, but ideally the caller should ensure it has an event instance.